lib/main/syncService.ts

```diff
@@ -236,6 +242,36 @@ export class SyncService {
       }
     }
   }
+
+  private async syncAdvancedSettings(lastSyncedAt?: string) {
+    try {
+      // Get remote advanced settings
+      const remoteSettings = await grpcClient.getAdvancedSettings()
+
+      // Compare timestamps to determine sync direction
+      const remoteUpdatedAt = new Date(remoteSettings.updatedAt)
+      const lastSyncTime = lastSyncedAt ? new Date(lastSyncedAt) : new Date(0)
+
+      // If remote settings were updated after last sync, pull them to local
+      if (remoteUpdatedAt > lastSyncTime) {
+        const updatedLocalSettings: AdvancedSettings = {
+          llm: {
+            asrModel: remoteSettings.llm?.asrModel || 'whisper-large-v3',
+          },
+        }
+
+        // Update local store
+        mainStore.set(STORE_KEYS.ADVANCED_SETTINGS, updatedLocalSettings)
```

Comment:
The paradigm I was originally thinking was that anything that needs to be synced to the server should live in sqlite. I think we should keep to this paradigm unless you think there's a good reason to switch now.

Reply:
Yeah we could put advanced settings in a sqlite table but my reason to keep it just at the store and not also in a sqlite table is because it is not repeated data. There should only be one copy of a user's advanced settings, and that will have to be fetched from the store to display in the renderer.